### ISSUE TITLE
programmatically associated is not explained in "text alternative" definition

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag3#498]
https://deploy-preview-414--wcag3.netlify.app/guidelines/#dfn-text-alternative

> > text alternative
> > text that is [programmatically associated] with non-text content or referred to from text that is programmatically associated with non-text content

In WCAG 2.0 this term is explained with the following sentence, but for some reason that sentence was dropped from WCAG 3.0. I don't think it's right to just link to the programmatically determined definition and hope people will understand what that means because they read it in WCAG 2. WCAG 3.0 needs to be self-explanatory.

> Programmatically associated text is text whose location can be [programmatically determined] from the non-text content.

I don't think WCAG 3.0 should have this be part of the definition, for two reaons:

1. Firstly I question whether this actually matters. The way this has worked in practice is that if an image has information, and that information is also available in a nearby text, that image is just considered decorative. I'm not aware of anyone who for example of an icon with a text next to it, who'd fail that if the image didn't have aria-labelledby pointing at the text. I don't think anything really changes if we remove this part of the definition.

2. If we do want to keep this, I don't think this belongs in a definition. This is really just a hidden requirement. Text alternatives need to be programmatically associated with non-text content. That can be its own requirement, or it can be part of the current one, but I think by putting it in the definition WCAG 2.0 has kind of hidden that this actually is required.
